### DISCUSSION TITLE
Transferring parameters after multi domain calibration

### DISCUSSION THREAD

**Discussion by bibek-ux** [mhm-ufz/mHM#107] [Q&A] [answered]
Help needed!
I have obtained an optimized set of parameters by multidomain calibration with KGE values for 8 domain as follows optimized at 0.25 degrees resolution. Each domain contained sub basins from our catchment area at outlet of which had a gauging station.
domain 1 = 0.900
domain 2 = 0.78
domain 3 = 0.7
domain 4 = 0.9
domain 5 = 0.88
domain 6 = 0.71
domain 7 = 0.8
domain 8 - 0.9
Next I used these optimized parameter sets to run the model in a single domain using dem masked for our entire catchment area. I included all the 8 gauges in this domain and ran the model at 0.03125 degrees but got KGE values as follows
Gauge 1 = 0.9
Gauge 2 = 0.41
Gauge 3 = 0.07
Gauge  4  = 0.90
Gauge 5 = 0.048
Gauge 6 = 0.71
Gauge 7 = 0.80
Gauge 8 = 0.22
Am I missing something here ? Shouldn't be getting same KGE values as before?
Thanks

**Comment by pallavhydro** [comment, marked as answer]
Hi Bibek

The 0.25 multi-domain calibration looks awesome!

As you know, mHM is a grid based model. Which means, while routing, depending upon model resolution,  the area of small catchments are not conserved well with `single DEM`. That is why we go the multi-domain way, as in your calibration stage. While you are transferring the optimised parameters to another scale, which is possible with mHM due to its scalability feature, you are but using `single DEM` in this second stage. Although 0.03125 is much finer than 0.25, it seems some of your gauges catchments were still too small for 0.03125 and thus not conserved well-enough e.g., gauges 3 and 5. 

Solutions:
1. Either go further down to a finer resolution. The thumb rule: the grid area (1 pixel) should be at least 30 times smaller than the smallest gauge catchment you want to conserve. 
2. Else, in case streamflow (Q) is not your main application, which I am guessing is your case as you want model application with single DEM e.g., soil moisture, the streamflow's purpose would be limited to the multi-domain calibration for obtaining basin-wide mHM parameters, which you've obtained nicely in the first stage. In second stage (at 0.03125) one can just use the mHM water balance components like soil-moisture or ET, without worrying about Q which will remain poor for gauges 3 and 5.

This `small catchment problem` is common in streamflow simulations from all gridded hydrological models. The mHM development team would soon bring a novel stream network upscaling method (paper under review) that would resolve the issue entirely. Stay tuned for it!

Hope this helps.

Pallav

**Reply by bibek-ux** [reply]
Thank you so much Pallav dai, 

 I have been stressing over this for days. You are right the mHM_fluxes (soil-moisture) was matching very nicely with remotely sensed soil moisture products, but the single domain KGE was all over the place. I did have a hunch that "small catchment" problem was the issue (as the problem was even bigger in larger resolutions) but could not conclude anything. I will next try running the model at even finer resolutions. 

Thanks again.
